### ISSUE TITLE
MB-MelGAN weight normalization doesn't converge

### ISSUE THREAD

**Issue by MMingabc** [TensorSpeech/TensorFlowTTS#681]
Hi,

I'm trying to apply weight normalization to MB-MelGAN's generator. 
But it doesn't converge even during the pre-training phase.
How can I fix this problem?
According to MelGAN paper, normalization is _extremely crucial_. Is this true for other datasets?

Thank you.

**Comment by dathudeptrai** [comment]
@MMingabc The weight normalization in TF is something wrong :D. BTW, the performance of melgan without weight norm is still ok :D

**Comment by MMingabc** [comment]
@dathudeptrai Thank you! I just replaced TFTTS's built-in weight normalization by TF-addon's implementation. MB-MelGAN has started to converge. I will update the result after finishing training.

**Comment by dathudeptrai** [comment]
@MMingabc I think the implementation is similar but maybe they already fix some bugs :D

**Comment by GaryGao99** [comment]
It will be converge when set  "data_init=False" in class "WeightNormalization", and faster converge if replace "tf.keras.initializers.GlorotNormal(seed=initializer_seed)"
 with 
"tf.random_normal_initializer(mean=0.0, stddev=0.02, seed=initializer_seed)" 
in function "get_initializer" .
